provider/win_driver_utils.py

```diff
@@ -211,15 +212,16 @@ def run_installer(vm, session, test, params, run_installer_cmd):
             params, session), 240, 2, 2):
         test.fail("Autoit exe stop there for 240s,"
                   " please have a check.")
-
-    if cdrom_virtio_version in VersionInterval(installer_restart_version):
+    restart_con_ver = cdrom_virtio_version in VersionInterval(
+        installer_restart_version)
+    restart_con_repair = "repair" in run_installer_cmd
+    if restart_con_ver or restart_con_repair:
+        # Wait for vm re-start by installer itself
         if not utils_misc.wait_for(lambda: not session.is_responsive(),
                                    120, 5, 5):
             test.fail("The previous session still exists,"
                       "seems that the vm doesn't restart.")
         session = vm.wait_for_login(timeout=360)
-    else:
```

Comment:
Hi @xiagao 

Would you please update these 2 lines code as we discussed.Thanks!

Reply:
Updated. @leidwang